lib/config.js

```diff
@@ -28,10 +27,12 @@ const defaults = {
 };
 
 function loadConfigs(path) {
-  const configPath = path || Path.resolve(__dirname, '../config/dev.json');
-  const configs = nconf.file(configPath);
+  let configs = nconf.defaults(defaults);
+
+  if (path) {
+    configs = nconf.file(path);
+  }
 
-  configs.defaults(defaults);
```

Comment:
Looks like we lost the behavior where defaults got filled in for config bits that weren't in the file. Was that intentional?